### ISSUE TITLE
Small tweak to get working on Ubuntu

### ISSUE THREAD

**Issue by jgrn307** [apburt/treeseg#1]
I was able to get this working on Ubuntu, but there is an issue noted at:
https://github.com/PointCloudLibrary/pcl/issues/1594

I was able to modify your instructions for RHEL to get treeseg working as follows:

apt-get install -y git cmake
apt-get install -y libpcl-dev
apt-get install -y libproj-dev # Note that proj4 appears to be a requirement of treeseg
ln -s /usr/lib/x86_64-linux-gnu/libvtkCommonCore-6.2.so /usr/lib/libvtkproj4.so # this is the tweak
git clone https://github.com/apburt/treeseg.git;
cd treeseg;
mkdir build;
cd build;
cmake ../src;
make;

In a future release, could you consider searching for libvtkproj4.so as listed above?  Otherwise the symlink trick will work fine.  Cheers!

**Comment by chaofeng2007** [comment]
@jgrn307 

Have you run treeseg successfully?

I seems that I need to strictly follow this order of the Usage to segment orest point cloud:

> plotcoords ../matrix/ > nouragesH20_coords.dat
rxp2pcd ../ nouraguesH20_coords.dat 60 15 nouraguesH20
nearestneighbour 1 4 nouraguesH20.sample.pcd > nouraguesH20_nn.dat
downsample 0.04 1 nouraguesH20_*.pcd
getdemslice 2 3 6 nouraguesH20_*.downsample.pcd
findstems 15 0.2 2 ../nouraguesH20_coords.dat ../nouraguesH20.slice.downsample.pcd
segmentstem 12.5 ../nouraguesH20.downsample.pcd ../clusters/cluster_*.pcd
getcrownvolume ../nouraguesH20.downsample.pcd ../stems/stem_*.pcd
segmentcrown [14 - 16] ../volume_*.pcd

However, the download source does not contain `matrix`folder, `*.pcd`file. 

I tried to run `nearestneighbour 1 4 nouraguesH20.sample.pcd > nouraguesH20_nn.dat`, but I got this error `Segmentation fault (core dumped)`.

How can I fix it?

Thank you.
